source/scenes/Error/index.jsx

```diff
@@ -1,6 +1,9 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import ReactGA from 'react-ga';
+ReactGA.initialize('UA-46010489-4'); //imports and initializes react packages for GA
```

Comment:
I'm not super familiar with ReactGA but it seems like you only need to initialize it once. I would move this statement into `source/App.jsx` and remove any other initializes from the code.